src/material/slider/slider-interface.ts

```diff
@@ -0,0 +1,215 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {InjectionToken} from '@angular/core';
+import {Thumb} from '@material/slider';
```

Comment:
Should we just declare our own enum for this? That way we can drop the dependency on MDC's JS completely.